api/endpoints/community.py

```diff
@@ -13,14 +12,14 @@ def make_blueprint(con):
     
     metric = CommunityMetrics(con)
     metric_rbu = RentBurdenedMetrics(con)
-    metric_tt = TaxiTripMetrics(con)
 
     supported_metrics = {
         "rideshare_pickups_covid": metric.rideshare_total_pickups,
         "rideshare_pooled_trip_rate_2018": lambda: metric.rideshare_pooled_trip_rate(year=2018),
         "rideshare_pooled_trip_rate_2019": lambda: metric.rideshare_pooled_trip_rate(year=2019),
         "rideshare_pool_request_rate_2018": lambda: metric.rideshare_pool_request_rate(year=2018),
         "rideshare_pool_request_rate_2019": lambda: metric.rideshare_pool_request_rate(year=2019),
+        "total_population_2000": lambda: metric.population(year=2000, segment="all"),
```

Comment:
Remove this line, we had to remove this metric because Chicago Health Atlas removed the data for it.

I think you started your branch in an older state where it still existed, which is why you are adding it back.